packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -1232,6 +1232,9 @@ export async function categorizePackageDirs(packageDirs: string[]) {
           if (!role) {
             return; // Ignore packages without roles
           }
+          if (pkgJson?.backstage?.inline) {
+            return; // Ignore inline packages
```

Comment:
Why exclude these? They still have an actual surface that's interesting to see and track, right?

Reply:
Hmm, yeah I guess it could help out with figuring out the impact of a change too. I'll check if we can include them

Reply:
Updated with a comment, the issue is that `@internal`s can't be exported as it crashes API extractor, but most of the time that's what we want them to be marked as. I don't think it makes sense to mark these exports with `@public` etc, because they generally shouldn't be. I updated the comment to provide a bit more info and made it a TODO, since it would be nice if we could have API reports for inline packages too.